vncviewer/parameters.cxx

```diff
@@ -64,6 +64,9 @@ BoolParameter emulateMiddleButton("EmulateMiddleButton",
 BoolParameter dotWhenNoCursor("DotWhenNoCursor",
                               "Show the dot cursor when the server sends an "
                               "invisible cursor", false);
+BoolParameter largeCursorInsteadOfDot("LargeCursorInsteadOfDot",
+                                      "Use a large cursor bitmap instead of a dot "
+                                      "when using the dot cursor", false);
```

Comment:
Having multiple boolean options feels confusing. I think this is now a multiple-choice question, where we pick between "no cursor", "dot cursor", or "pointer cursor".